/thrust_allocator:
  ros__parameters:
    num_thrusters: 4
    t1:
      contrib: 
        x: 1
      ly: 1.0
    t2:
      contrib: 
        x: 1
      ly: 1.0
    t3:
      contrib: 
        y: 1
      lx: 1.0
    t4:
      contrib: 
        y: 1
      lx: 1.0